stan/math/gpu/opencl_context.hpp

```diff
@@ -133,6 +140,9 @@ class opencl_context_base {
          {"LOWER_TO_UPPER", static_cast<int>(TriangularMapGPU::LowerToUpper)},
          {"THREAD_BLOCK_SIZE", 32},
          {"WORK_PER_THREAD", 8}};
+  // TODO(Steve): Make these tunable during warmup
+  map_base_opts tuning_opts_ = {
```

Comment:
My naive thought here is that this doesn't need to be a map, because we're not going to dynamically add and remove keys from this - like we expect these things to always exist and we'd rather have our spelling of e.g. `cholesky_size` checked by the compiler.

Reply:
Yeah I agree with that, I'll turn this into a struct